src/components/Footer/Footer.js

```diff
@@ -13,13 +13,14 @@ const StyledFooter = styled.footer`
 const StyledIcon = styled(NavLink)`
   text-decoration: none;
   color: #fcfbf6;
-  font-size: 40px;
+  font-size: 35px;
```

Comment:
I am not sure, but maybe a variable font-size could be possible too, like 0.8em. just an idea